LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/Region.cs

```diff
@@ -354,6 +354,11 @@ public DataRateIndex GetDRFromFreqAndChan(string datr)
             return DRtoConfiguration.FirstOrDefault(x => x.Value.DataRate.XpkDatr == datr).Key;
         }
 
+        public DataRateIndex GetDataRateIndex(DataRate datr)
```

Comment:
This seems to be only used in tests so either we should move into an extension method in the common tests project or add unit tests for it. I think the former might be easier if this is just temporary?

Reply:
Good idea

Reply:
I am not sure, I think it might be also used in the next PR, I ill move it back